src/main/scala/com.snowplowanalytics.s3/loader/KinesisSourceExecutor.scala

```diff
@@ -53,12 +54,18 @@ class KinesisSourceExecutor(
   badSink: ISink,
   serializer: ISerializer,
   maxConnectionTime: Long,
-  tracker: Option[Tracker[Id]]
+  tracker: Option[Tracker[Id]],
+  disableCloudWatch: Option[Boolean]
```

Comment:
Isn't `cloudWatchEnabled: true / false` more intuitive?

Also, does it need to be an `Option`? Why not a simple Boolean with default value?

Reply:
Good point !